### PR TITLE
Extend documentation of the editor

### PR DESCRIPTION
**Pull request by blcham**

### CODE REVIEW

**Comment by blcham** [comment]
@palagdan please check

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # SPipes Editor UI
 
+SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:
+- visualization of the scripts
+  - exploration of the related filesystem structure where scripts are located
```
Comment (palagdan) [inline README.md:5]: Maybe the following:

`browse the related filesystem structure where the scripts are stored`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+  - represent the pipelines as graphs
+  - collapse/expand groups of nodes based on the source file they come from
+  - automatic layout of graph nodes
+- modification of the scripts
```
Comment (palagdan) [inline README.md:9]: Maybe the following:

`script Modification`

File: README.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+  - notification if script file changes outside of the editor
```
Comment (palagdan) [inline README.md:10]: Maybe the following:

`receive notifications if a script file changes outside the editor`

File: README.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+  - custom editors for specific modules (e.g. SPARQL query editor)
+  - editor to edit ttl directly
```
Comment (palagdan) [inline README.md:12]: Maybe thee following:

`directly edit .ttl files using a built-in editor.`

File: README.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
+- execution of the scripts
+  - execute scripts with parameters
```
Comment (palagdan) [inline README.md:14]: Maybe the following:

`execute scripts with specified parameters`

File: README.md [hunk 6]
```diff
@@ -1,5 +1,19 @@
...
 
+SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:
+- visualization of the scripts
+  - exploration of the related filesystem structure where scripts are located
+  - represent the pipelines as graphs
+  - collapse/expand groups of nodes based on the source file they come from
+  - automatic layout of graph nodes
+- modification of the scripts
+  - notification if script file changes outside of the editor
+  - custom editors for specific modules (e.g. SPARQL query editor)
```
Comment (palagdan) [inline README.md:11]: Maybe the following:

`use custom editors for specific modules (e.g., an SPARQL query editor)`

File: README.md [hunk 7]
```diff
@@ -1,5 +1,19 @@
...
+  - editor to edit ttl directly
+- execution of the scripts
+  - execute scripts with parameters
+  - explore the history of executions
```
Comment (palagdan) [inline README.md:15]: Maybe the following:

`browse the history of previous executions`

File: README.md [hunk 8]
```diff
@@ -1,5 +1,19 @@
 # SPipes Editor UI
 
+SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:
+- visualization of the scripts
+  - exploration of the related filesystem structure where scripts are located
+  - represent the pipelines as graphs
+  - collapse/expand groups of nodes based on the source file they come from
```
Comment (palagdan) [inline README.md:7]: Maybe the following:

`collapse/expand groups of nodes based on the source file`

File: README.md [hunk 9]
```diff
@@ -1,5 +1,19 @@
 # SPipes Editor UI
 
+SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:
+- visualization of the scripts
+  - exploration of the related filesystem structure where scripts are located
+  - represent the pipelines as graphs
```
Comment (palagdan) [inline README.md:6]: Maybe the following:

`represent pipelines as interactive graphs`

File: README.md [hunk 10]
```diff
@@ -1,5 +1,19 @@
...
+  - collapse/expand groups of nodes based on the source file they come from
+  - automatic layout of graph nodes
```
Comment (palagdan) [inline README.md:8]: Maybe the following:

`automatically arrange graph nodes`

File: README.md [hunk 11]
```diff
@@ -1,5 +1,19 @@
 # SPipes Editor UI
 
+SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:
+- visualization of the scripts
```
Comment (palagdan) [inline README.md:4]: Maybe the following:

`script visualization`

**Review by palagdan — COMMENTED** [review]
See my comments

File: README.md (hunk 11 above)
Comment (palagdan) [inline README.md:4]:
```suggestion
- script visualization
```

File: README.md (hunk 1 above)
Comment (palagdan) [inline README.md:5]:
```suggestion
  - browse the related filesystem structure where the scripts are stored
```

File: README.md (hunk 9 above)
Comment (palagdan) [inline README.md:6]:
```suggestion
  - represent pipelines as interactive graphs
```

File: README.md (hunk 8 above)
Comment (palagdan) [inline README.md:7]:
```suggestion
  - collapse/expand groups of nodes based on the source file
```

File: README.md (hunk 10 above)
Comment (palagdan) [inline README.md:8]:
```suggestion
  - automatically arrange graph nodes
```

File: README.md (hunk 2 above)
Comment (palagdan) [inline README.md:9]:
```suggestion
- script Modification
```

File: README.md (hunk 3 above)
Comment (palagdan) [inline README.md:10]:
```suggestion
  - receive notifications if a script file changes outside the editor
```

File: README.md (hunk 6 above)
Comment (palagdan) [inline README.md:11]:
```suggestion
  - use custom editors for specific modules (e.g., an SPARQL query editor)
```

File: README.md (hunk 4 above)
Comment (palagdan) [inline README.md:12]:
```suggestion
  - directly edit .ttl files using a built-in editor.
```

File: README.md (hunk 5 above)
Comment (palagdan) [inline README.md:14]:
```suggestion
  - execute scripts with specified parameters
```

File: README.md (hunk 7 above)
Comment (palagdan) [inline README.md:15]:
```suggestion
  - browse the history of previous executions
```

**Review by palagdan — COMMENTED** [review]
See my comments

File: README.md (hunk 7 above)
Comment (palagdan) [inline README.md:15]:
```suggestion
  - browse the history of previous executions
```

File: README.md (hunk 11 above)
Comment (palagdan) [inline README.md:4]:
```suggestion
- script visualization
```

File: README.md [hunk 12]
```diff
@@ -1,5 +1,19 @@
...
+- script visualization
+  - browse the related filesystem structure where the scripts are stored
+  - represent pipelines as interactive graphs
+  - collapse/expand groups of nodes based on the source file
+  - automatically layout graph nodes
+- script modification
+  - receive notifications if a script file changes outside the editor
+  - use custom editors for specific modules (e.g., an SPARQL query editor)
+  - directly edit .ttl files using a built-in editor
+- execution of the scripts
```
Comment (blcham) [inline README.md:13]:
```suggestion
-  script execution
```

**Comment by blcham** [comment]
@palagdan, thanks for the review; the suggestions were very good!

**Pull request merged**